openapi/components/schemas/GatewayAccountConfig/Ilixium.yaml

```
@@ -45,4 +45,11 @@ allOf:
               - itix
               - tpg
       threeDSecureServer:
-        $ref: ../ThreeDSecureServers/Ilixium3dsServers/Ilixium3dsServers.yaml
+        type: object
+        description: Ilixium 3DS servers.
```

question: should this be singular or plural
```suggestion
        description: Ilixium 3DS server.
```